(CAAPP-431) Upgraded connectivity_plus Package

klortiz13 opened the pull request:
## Summary
Upgraded connectivity plus package. No breaking changes.
I also had to upgrade Kotlin to run the Android emulator on VS Code and remove warnings. No breaking changes.

## Changelog
[General] [Change] - From `6.1.0` to `7.0.0`


## Test Plan
Wifi card is the only card that uses this. Ensure nothing has changed.

[Test plan for Android](https://ucsdcloud.sharepoint.com/:x:/r/sites/WorkplaceTechnologyServices-CampusMobileBuilds/_layouts/15/Doc.aspx?sourcedoc=%7B1BFB14CB-8A31-4BCE-894B-4B2BBF71DD6B%7D&file=PR-2273-Test-Plan-7.34.0-QA-4832.xlsx&action=default&mobileredirect=true)
[Test plan for iOS
](https://ucsdcloud.sharepoint.com/:x:/r/sites/WorkplaceTechnologyServices-CampusMobileBuilds/_layouts/15/Doc.aspx?sourcedoc=%7BB4D6B4BB-09C6-465C-B5F3-F54414D27ECA%7D&file=PR-2273-Test-Plan-7.34.0-QA-4831.xlsx&action=default&mobileredirect=true)

klortiz13 committed:
VS Code automatically added files to run Android emulator

klortiz13 committed:
removed android build warnings by upgrading kotlin to 2.1.0. No breaking changes

klortiz13 committed:
Upgraded connectivity_plus

klortiz13 committed:
removed the ^ character to keep connectivity plus at 7.0.0

hamedhemmati reviewed (changes requested):
This is changing the build.gradle and changing the minSdk to flutter's internal minSdkVersion that won't be compatible with the Wifi package. Any reason why build.gradle was updated?

klortiz13 commented:
> This is changing the build.gradle and changing the minSdk to flutter's internal minSdkVersion that won't be compatible with the Wifi package. Any reason why build.gradle was updated?

<img width="1546" height="551" alt="image" src="https://github.com/user-attachments/assets/eb1dc3b7-4cab-414e-a2c7-2467ed7830e2" />
For some reason VS Code keeps changing this every time I run `flutter run`. I forgot to change it back for this PR. Thanks for catching that!

klortiz13 committed:
Updated build.gradle.kts

The new files were added by VS Code so Android emulators can run. I used to use android studio (which didn't require these files) but I am using VS Code now. That is why these files were added.
